sdk/devcenter/Azure.Developer.DevCenter/src/Generated/AzureDeveloperDevCenterClientOptions.cs

```diff
@@ -24,8 +24,8 @@ public enum ServiceVersion
 
         internal string Version { get; }
 
-        /// <summary> Initializes new instance of DevCenterClientOptions. </summary>
-        public DevCenterClientOptions(ServiceVersion version = LatestVersion)
+        /// <summary> Initializes new instance of AzureDeveloperDevCenterClientOptions. </summary>
+        public AzureDeveloperDevCenterClientOptions(ServiceVersion version = LatestVersion)
```

Comment:
This change was actually caused by moving to TypeSpec. Because our namespace is Azure.Developer.DevCenter the client option get updated to `AzureDeveloperDevCenterClientOptions`